src/setup/kmd/mod.rs

```diff
@@ -184,6 +188,31 @@ impl Kmd {
 
         Err(anyhow!("the kmd instance is not started"))
     }
+
+    /// Get the list of public keys in the wallet.
+    pub async fn get_keys(&mut self, wallet_handle: String) -> anyhow::Result<ListKeysResponse> {
+        if let Some(rest_client) = &self.rest_client {
```

Comment:
Why no `match` here?

Reply:
I dunno, I find it more readable than this, which is more tightly coupled:
```rust
match self.rest_client {
  Some(rest_client) => rest_client.get_keys(wallet_handle).await,
  _ => Err(anyhow!("the kmd instance is not started")),
}
```